From 0680b6bf003ecd685be34c516a1dd8b85616770c Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Sun, 29 Nov 2015 21:22:30 -0500
Subject: [PATCH 0772/1936] PADnotePars: Squelch 'needPrepare' Unknown Address

Future versions may provide a precise knowledge of when an
applyparameters() is needed, but for now no such information is stored.
Adding a dummy port silences this one runtime warning.
---
 src/Params/PADnoteParameters.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/Params/PADnoteParameters.cpp b/src/Params/PADnoteParameters.cpp
index 0d12efab..14e5d9b9 100644
--- a/src/Params/PADnoteParameters.cpp
+++ b/src/Params/PADnoteParameters.cpp
@@ -234,6 +234,8 @@ static const rtosc::Ports non_realtime_ports =
             d.reply(d.loc, "b", n*sizeof(float), tmp);
             d.reply(d.loc, "i", realbw);
             delete[] tmp;}},
+    {"needPrepare:", rDoc("Unimplemented Stub"),
+        NULL, [](const char *, rtosc::RtData&) {}},
 };
 
 const rtosc::Ports &PADnoteParameters::non_realtime_ports = ::non_realtime_ports;
-- 
2.47.0

